metricflow/test/snapshots/test_cumulative_metrics.py/str/BigQuery/test_grain_to_date_cumulative_metric__query_output.txt

```diff
@@ -1,5 +1,5 @@
   metric_time__day  revenue_mtd
-0       2021-01-03         1000
-1       2021-01-04         2000
-2       2021-01-05         2000
-3       2021-01-06         2000
+0       2021-01-03         3000
```

Comment:
These updated numbers are correct, while the old ones were missing inputs. See https://github.com/dbt-labs/metricflow/blob/main/metricflow/test/fixtures/source_table_snapshots/simple_model/fct_revenue.yaml#L15-L18